p2p/src/net/libp2p/mod.rs

```diff
@@ -368,6 +368,19 @@ where
         &self.peer_id
     }
 
+    async fn ban_peer(&mut self, peer_id: T::PeerId) -> crate::Result<()> {
+        log::debug!("ban peer {}", peer_id);
```

Comment:
Perhaps it's helpful to print more information. At least the ip address and the reason for banning. The peer_id is meaningless without context.

Reply:
I'll log that information in `PeerManager`